facefusion/core.py

```diff
@@ -92,7 +89,7 @@ def pre_check() -> bool:
 
 
 def common_pre_check() -> bool:
-	return content_analyser.pre_check() and face_analyser.pre_check() and face_masker.pre_check() and voice_extractor.pre_check()
+	return content_analyser.pre_check() and face_classifier.pre_check() and face_detector.pre_check() and face_landmarker.pre_check() and face_masker.pre_check() and face_recognizer.pre_check() and voice_extractor.pre_check()
```

Comment:
it looks very long, something like this would be nice
```python
def common_pre_check() -> bool:
    components = [
        content_analyser,
        face_classifier,
        face_detector,
        face_landmarker,
        face_masker,
        face_recognizer,
        voice_extractor
    ]
    return all(component.pre_check() for component in components)
```